src/net/rpc.rs

```diff
@@ -163,6 +166,38 @@ impl PeerInfoData {
     }
 }
 
+impl RPCNeighborsInfo {
+    /// Load neighbor address information from the peer network
+    pub fn from_p2p(network_id: u32, peers: &PeerMap, chain_view: &BurnchainView, peerdb: &PeerDB) -> Result<RPCNeighborsInfo, net_error> {
+        let neighbor_sample = PeerDB::get_random_neighbors(peerdb.conn(), network_id, MAX_NEIGHBORS_DATA_LEN, chain_view.burn_block_height, false)
+            .map_err(net_error::DBError)?;
+
+        let sample : Vec<RPCNeighbor> = neighbor_sample
+            .iter()
```

Comment:
You can avoid a clone here by using `into_iter`

Reply:
Fixed in 5dda11c